aries-site/src/pages/templates/global-banner-notifications.mdx

```diff
@@ -43,21 +44,24 @@ For low-severity notifications that do not require user action or user attention
 ## Anatomy
 
 <Example
-  caption="Diagram illustrating the four parts composing of a Banner Notification."
+  caption="Diagram illustrating the four parts of a Global Notification."
   plain
 >
   <BannerNotificationDiagram />
 </Example>
 
-1. **Status Indicator** is an icon made of a shape and color to capture the user’s attention.
-2. **Message** is a short description explaining the issue.
-3. **Anchors (optional)** are used to direct users to links related to the notification.
-4. **Close Button (optional)** is used to dismiss the notification.
+| Label | Region                       | Purpose                                                      |                  Required                   | Notes                                                                                                                  |
+| ----- | ---------------------------- | ------------------------------------------------------------ | :-----------------------------------------: | ---------------------------------------------------------------------------------------------------------------------- |
+| **1** | **Icon or status indicator** | Indicates the status of notification                         | <Checkmark a11yTitle="true" size="small" /> | <span>Warning, Critical, and Unknown [status indicators](/templates/status-indicator) can be displayed.</span>         |
```

Comment:
Can you add `--` for the empty cells? Similar to: https://deploy-preview-3759--keen-mayer-a86c8b.netlify.app/components/layer#anatomy